app/views/account/weeklyRenew.scala.html

```diff
@@ -62,7 +63,7 @@ <h3 class="mma-section__header">Renew your subscription</h3>
                 data-billing-country="@billToCountry.alpha2"
                 data-promo-code="@promoCode.map(_.get)"
                 data-currency="@currency.iso"
-                data-delivery-country="@contact.mailingCountryParsed.map(_.alpha2).getOrElse(billToCountry.alpha2)"
+                data-delivery-country="@contact.country.map(_.alpha2).getOrElse(billToCountry.alpha2)"
```

Comment:
it's only optional in the reader in case we don't have it in our lookup/enum.  I think in reality it should be there, but we would end up with a more fail fast situation as the whole sub would be unreadable in that case (as it would either throw an exception or return a Left.)